rasa/shared/core/domain.py

```diff
@@ -309,8 +311,93 @@ def merge_lists_of_dicts(
 
         return self.__class__.from_dict(combined)
 
+    @staticmethod
+    def merge_domain_dicts(
+        domain1: Dict, domain2: Dict, override: bool = False
+    ) -> Dict[Text, Any]:
+        """Merge this domain dict with another one, combining their attributes.
```

Comment:
```suggestion
        """Merges this domain dict with another one, combining their attributes.
```

Reply:
I changed it to the style that we prefer in the docs (can't seem to find the reference for it on Notion...)